com.unity.formats.fbx/Editor/Unity.Formats.Fbx.Editor.api

```diff
@@ -0,0 +1,125 @@
+        public virtual UnityEngine.Transform AnimationDest { get; }
+        public virtual UnityEngine.Transform AnimationSource { get; }
+        public virtual bool EmbedTextures { get; }
+        public virtual UnityEditor.Formats.Fbx.Exporter.ExportFormat ExportFormat { get; }
+        public abstract bool ExportUnrendered { get; }
+        public virtual bool KeepInstances { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.LODExportType LODExportType { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.Include ModelAnimIncludeOption { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.ObjectPosition ObjectPosition { get; }
+        public virtual bool PreserveImportSettings { get; }
+        public virtual bool UseMayaCompatibleNames { get; }
+        protected ExportOptionsSettingsSerializeBase() {}
+        public virtual bool Equals(object e);
+        public virtual int GetHashCode();
+        public void SetAnimatedSkinnedMesh(bool animatedSkinnedMesh);
+        public void SetAnimationDest(UnityEngine.Transform dest);
+        public void SetAnimationSource(UnityEngine.Transform source);
+        public void SetExportFormat(UnityEditor.Formats.Fbx.Exporter.ExportFormat format);
+        public void SetUseMayaCompatibleNames(bool useMayaCompNames);
+    }
+
+    public interface IExportOptions
+    {
+        public abstract bool AllowSceneModification { get; }
+        public abstract bool AnimateSkinnedMesh { get; }
+        public abstract UnityEngine.Transform AnimationDest { get; }
+        public abstract UnityEngine.Transform AnimationSource { get; }
+        public abstract bool EmbedTextures { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.ExportFormat ExportFormat { get; }
+        public abstract bool ExportUnrendered { get; }
+        public abstract bool KeepInstances { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.LODExportType LODExportType { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.Include ModelAnimIncludeOption { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.ObjectPosition ObjectPosition { get; }
+        public abstract bool PreserveImportSettings { get; }
+        public abstract bool UseMayaCompatibleNames { get; }
+    }
+
+    public enum Include
+    {
+        public const UnityEditor.Formats.Fbx.Exporter.Include Anim = 1;
+        public const UnityEditor.Formats.Fbx.Exporter.Include Model = 0;
+        public const UnityEditor.Formats.Fbx.Exporter.Include ModelAndAnim = 2;
+        public int value__;
+    }
+
+    public enum LODExportType
+    {
+        public const UnityEditor.Formats.Fbx.Exporter.LODExportType All = 0;
+        public const UnityEditor.Formats.Fbx.Exporter.LODExportType Highest = 1;
+        public const UnityEditor.Formats.Fbx.Exporter.LODExportType Lowest = 2;
+        public int value__;
+    }
+
+    public sealed class ModelExporter : System.IDisposable
```

Comment:
Is this the time when we can finally get rid of IDisposable here? We aren't actually disposable, it's all static.